### PR TITLE
Wire payments fintech dotnet changes

### PR DESCRIPTION
**Pull request by natashasehgal**
Add new version 3.5.0 corresponding to new API version 206. This PR has Wire complex payment method added

Testing:
./build.sh executed tasks clean, restore, generate, build, test successfully

### CODE REVIEW

File: Sift/Schema/ComplexTypes/wire.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Wire",
+  "type": "object",
+  "required": [ "$wire_type", "$account_holder_name"],
```
Comment (natashasehgal) [inline Sift/Schema/ComplexTypes/wire.json:5]: @rsomavarapu-sift 

There's an error in the generate step if I add
```
  "oneOf": [
    {
      "required": [
        "$routing_number"
      ]
    },
    {
      "required": [
        "$swift_id"
      ]
    },
    {
      "required": [
        "$bic"
      ]
    }
  ],
```
oneOf is present in code repo (java) and passes using [validator](https://www.jsonschemavalidator.net/). dotnet is having an issue understanding this. 

Including oneOf causes ./build.sh to fail at the generate step where schemas are generated

Comment (rsomavarapu-sift) [reply Sift/Schema/ComplexTypes/wire.json:5]: what's the error? We do the validation before ingestion in our API service, so this is not mandatory but good to see if we can do the validation here.

Comment (rsomavarapu-sift) [inline Sift/Schema/ComplexTypes/wire.json:5]: `routing_number` is also a required field. you can add it here.

**Review by rsomavarapu-sift — CHANGES_REQUESTED** [review]

File: Sift/Schema/ComplexTypes/wire.json (hunk 1 above)
Comment (natashasehgal) [reply Sift/Schema/ComplexTypes/wire.json:5]: Okay. There is no error, but I was not able to add the same validation in yaml

Comment (natashasehgal) [reply Sift/Schema/ComplexTypes/wire.json:5]: Customer should send one required -  either routing or swift or bic

Comment (rsomavarapu-sift) [reply Sift/Schema/ComplexTypes/wire.json:5]: oh yes, i missed that.

Comment (rsomavarapu-sift) [reply Sift/Schema/ComplexTypes/wire.json:5]: Yes, the 3rd party framework that we use doesn't support that: https://github.com/JamesNK/Newtonsoft.Json/issues/964.

**Review by rsomavarapu-sift — APPROVED** [review]
LGTM

**Pull request merged**